ambari-server/src/main/resources/stacks/stack_advisor.py

```diff
@@ -1656,7 +1667,7 @@ def recommendConfigurationsForSSO(self, services, hosts):
 
     # If recommendation for config groups
     if "config-groups" in services:
-      self.recommendConfigGroupsConfigurations(recommendations, services, components, hosts,
+      self.recommendConfigGroupsConfigurations(recommendations, services, componentsList, hosts,
```

Comment:
camel case used in python only for class names, in other cases it supposed to be like `component_list`